maze-info.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/activity/activity.info b/activity/activity.info
+index abb1d02..76aba63 100644
+--- a/activity/activity.info
++++ b/activity/activity.info
+@@ -3,9 +3,14 @@ name = Maze
+ exec = sugar-activity3 activity.MazeActivity
+ icon = activity-maze
+ activity_version = 29
++release_date = 2018-04-12
+ license = GPLv3+
++metadata_license = CC0-1.0
++update_contact = tch@sugarlabs.org
+ bundle_id = vu.lux.olpc.Maze
```

Comment:
Rename the bundle_id to `org.sugarlabs.Maze` here as well, bundle_id and app-id must be the same.

Reply:
Make sure we provide screenshots as well

Reply:
> Rename the bundle_id to `org.sugarlabs.Maze` here as well, bundle_id and app-id must be the same.

resolved in 35f15c9660bdf3057fdcebc1e9becb355440b1bf

Reply:
> Make sure we provide screenshots as well

resolved in 64d80a4b06ae5764b35a2ec60be9ce7b0e88a128